presto-native-execution/CMakeLists.txt

```diff
@@ -23,6 +24,21 @@ execute_process(
   OUTPUT_VARIABLE SCRIPT_CXX_FLAGS
   RESULT_VARIABLE COMMAND_STATUS)
 
+# Include velox's dependency macros
+# See velox/CMake/resolve_dependency_modules/README.md
+list(PREPEND CMAKE_MODULE_PATH "${VELOX_ROOT}/CMake"
+     "${VELOX_ROOT}/CMake/third-party")
+include(${VELOX_ROOT}/CMake/ResolveDependency.cmake)
```

Comment:
`include(ResolveDependency)` should work here because `CMAKE_MODULE_PATH `is set by previous line.